user.go

```diff
@@ -9,13 +9,27 @@ import (
 	"time"
 )
 
+// Mute represents an user mute
```

Comment:
```suggestion
// Mute represents a user mute
```

[Source](https://www.writersdigest.com/online-editor/a-before-consonants-and-an-before-vowels-is-not-the-rule). `an`/`a` is decided by the sound the letter makes. (Which gets confusing because in the US it would be `an herb` vs UK `a herb`.

Reply:
thanks